### PR TITLE
Made all examples use s3 credentials from App.config

### PR DESCRIPTION
**Pull request by krisis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,20 @@ $ mono Minio.Examples/ListBuckets.exe
...
+- Add your s3 credentials in `Minio.Examples/app.config` file
+
+- Build Minio solution
+
+- Move into `Minio.Examples/bin/Debug` directory (you could choose Release too, depending on which ones you build)
+
+- Compile the example as below:
+	`csc /r:Minio.dll /out:ListBuckets.exe "../../ListBuckets.cs"`
+
+- Copy `Minio.dll.config` to `ListBuckets.exe.config`
```
Comment (krishnasrinivas) [inline README.md]: where is Minio.dll.config in the repo?

Comment (krisis) [inline README.md]: In Minio.Examples/bin/{Debug/Release}.  The previous instruction is to move into that directory.  Let me know how I can improve the instructions to make it more obvious.

Comment (krishnasrinivas) [inline README.md]: Ah, does it get created during the step `Build Minio solution`? because i don't see Minio.Examples/bin/{Debug/Release}

Comment (krisis) [inline README.md]: Yes, it is created as part of build. Where is `Minio.Examples.dll` being created in your setup when you build the solution?

Comment (krishnasrinivas) [inline README.md]: I was looking at the github repo for `Minio.Examples/bin/{Debug/Release}` now I see that it's generated during.

File: Minio.Examples/ListBuckets.cs [hunk 2]
```diff
@@ -31,7 +32,8 @@ static int Main()
         {
           /// Note: YOUR-ACCESSKEYID and YOUR-SECRETACCESSKEY
           /// are dummy values, please replace them with original values.
-            var client = new MinioClient("s3.amazonaws.com", "YOUR-ACCESSKEYID", "YOUR-SECRETACCESSKEY");
```
Comment (harshavardhana) [inline Minio.Examples/ListBuckets.cs]: why not make this is a comprehensive change across all examples?

Comment (krisis) [inline Minio.Examples/ListBuckets.cs]: I will do that in this PR. I wanted to see if this approach seemed OK before converting examples en masse.

**Pull request merged**